forest/src/cli/chain_cmd.rs

```diff
@@ -82,17 +88,24 @@ impl ChainCommands {
 
                 let epoch = tipset.unwrap_or(chain_head.epoch());
 
-                let output_path = match output_path {
-                    Some(path) => path.to_owned(),
-                    None => {
-                        let now = OffsetDateTime::now_utc();
-                        format!(
-                            "forest_snapshot_{}-{}-{}_height_{}.car",
-                            now.year(),
-                            now.month(),
-                            now.day(),
-                            epoch,
-                        )
+                let now = OffsetDateTime::now_utc();
+
+                let month_string = format!("{:02}", now.month() as u8);
+                let year = now.year();
+                let day = now.day();
+                let chain_name = cfg.chain.name.clone();
```

Comment:
We need to ask the forest node for the chain name. Not sure if we have a RPC call for this or if we need to add one.

Reply:
Should not the chain name be the same as in the configuration?

Reply:
The forest node and the forest RPC can be run with different configurations. They can even be run on different machines.